doc/ssh_config/ssh_config.md

```diff
@@ -176,7 +176,7 @@ The ConfigDB will be extended with next objects:
 		"POLICIES":{
 			"authentication_retries": "6",
 			"login_timeout": "120",
-			"ports": "[22]",
+			"ports": "22",

```

Comment:
Why was this change reverted?

Reply:
I wasn't sure it was aligned with YANG model, now I think it is. Thanks